azure-pipelines.yml

```diff
@@ -13,6 +13,14 @@ steps:
 
 - script: |
     echo Configuring Environment
+    echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install 'system-images;android-27;google_apis;x86'
```

Comment:
can we have a variable for image name? Also, it might be useful to set platform and tools versions separately